includes/payments/actions.php

```diff
@@ -253,26 +253,31 @@ function edd_process_after_payment_actions( $payment_id = 0, $force = false ) {
 add_action( 'edd_after_payment_scheduled_actions', 'edd_process_after_payment_actions', 10, 1 );
 
 /**
- * Record payment status change
+ * Record order status change
  *
- * @since 1.4.3
- * @param int $payment_id the ID number of the payment
- * @param string $new_status the status of the payment, probably "publish"
- * @param string $old_status the status of the payment prior to being marked as "complete", probably "pending"
+ * @since 3.0
+ * @param string $old_status the status of the order prior to being marked as "complete", probably "pending".
+ * @param string $new_status the status of the order, probably "publish".
```

Comment:
```suggestion
 * @param string $new_status The new order status.
```

Better wording and `publish` is no longer used as a status.